generators/app/templates/python/README.md

```diff
@@ -1,29 +1,65 @@
 # <%= name %> modular service
 
-*<%= name %>* is a Viam modular service that provides <%= api_name_lower %> capabilities
+This module implements the [<%= api_namespace %> <%= api_name_lower %> API](https://github.com/<%= api_namespace %>/<%= api_name_lower %>-api) in a <%= model %> model.
+With this model, you can...
 
-The model this module makes available is <%= model %>
+## Requirements
 
-## Prerequisites
-
-Add instructions here for any prerequisites.
+_Add instructions here for any requirements._
 
 ``` bash
 ```
 
-## API
+## Build and Run
+
+To use this module, follow these instructions to [add a module from the Viam Registry](https://docs.viam.com/registry/configure/#add-a-modular-resource-from-the-viam-registry) and select the `<%= api_namespace %>:<%= api_name_lower %>:<%= model %>` model from the [`<%= model %>` module](https://app.viam.com/module/<%= api_namespace %>/<%= model %>).
 
-The <%= name %> resource implements the [<%= api_namespace %> <%= api_name_lower %> API](https://github.com/<%= api_namespace %>/<%= api_name_lower %>-api).
+## Configure your <%= api_name_lower %>
 
-Please also use the API codebase to interact with a configured version of this service via Viam SDK.
+> [!NOTE]  
+> Before configuring your <%= api_name_lower %>, you must [create a robot](https://docs.viam.com/manage/fleet/robots/#add-a-new-robot).
```

Comment:
Shouldn't we say machine now?

Reply:
Oh yes & probably should update the link, good catch